### ISSUE TITLE
Check approval process

### ISSUE THREAD

**Issue by krues8dr** [opengovfoundation/madison#719]
Currently, you can have an approved group but not be approved independently, and I'm not sure if you can create a document for the group under those circumstances (per the email this past week).  Verify that this works as expected.  We should also clean up this interface, so that you can create a document under each group (and have the proper group_id set as a result).

**Comment by krues8dr** [comment]
* [ ] Add message that you can't add members to a group that is `pending`

**Comment by cmbirk** [comment]
Throwing thoughts out here on this.

Currently the 'My Documents' page shows each group a user is in and what documents are owned by that group.  This included 'Independently Sponsored' documents.  I'm ok with that display, but could probably use a bit of design love.

If a group is pending the document list should read something like 'Pending groups cannot create documents.' in a gray, italic text.  The Group name should also be grey with a '(pending)' next to it.  This way if the user is expecting to see the group in their list it's clear they are in the group, but it's still waiting on verification.

There also should only be one create document button.  When the user clicks create document, they are given a modal with buttons for options that they can create the document as (ie. Independent, Group 1, etc. ), but not the pending groups.

When the button is clicked, it replaces the buttons with an input for a document title and a 'Create Document' button which would create the document and bring the user to the document editing page.  There should also be a '<< back' link to go back and select a different group.

**Comment by krues8dr** [comment]
I think there's friction in not allowing a user to take an action because their group is pending - after all, they shouldn't be beholden to us to interact.  For documents, this means that pending groups (and unverified users?) should be able to create documents, but not Publish them until they're verified/approved - this is dependent on #725 of course.

**Comment by cmbirk** [comment]
That makes sense to me, but let's open a separate ticket on that and prioritize this one since that's a somewhat different workflow.

**Comment by krues8dr** [comment]
:+1:

**Comment by krues8dr** [comment]
I made a quick patch there to add the group selection to the existing form, so this is at least not-broken for now, until we have time to address this properly.

**Comment by sethetter** [comment]
This is all good in v4 now!
